apps/src/music/views/topbuttons.module.scss

```diff
@@ -24,9 +24,45 @@
   }
 }
 
+.saveMessageContainer {
+  position: relative;
+}
+
+.saveMessage {
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 150px;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 5px;
+
+  &Saving {
+    color: $neutral-white;
+  }
+
+  &Error {
+    color: $red;
```

Comment:
this red is hard to read on the black background. Can we make it a lighter red?

Reply:
Updated!